.pre-commit-config.yaml

```diff
@@ -29,3 +29,8 @@ repos:
           - "--line-length"
           - "119"
         exclude: ^django_fsm_log/migrations|^docs/
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: "1.13.0"
+    hooks:
+    -   id: django-upgrade
+        args: [--target-version, "3.2"]
```

Comment:
This makes me think that we chose the "wrong" setting for ruff Pyupgrade: https://beta.ruff.rs/docs/settings/#target-version should be py37

It could be empty if we move some data from setup.py to pyproject.toml (would be another, bigger PR https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html)

Reply:
I agree.